### PR TITLE
update front-end for probe-search changes

### PR DESCRIPTION
**Pull request by spasovski**

### CODE REVIEW

File: src/components/search/Search.svelte [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <script>
 import { tick } from 'svelte';
-import { fly, fade } from 'svelte/transition';
+import { fade } from 'svelte/transition';
```
Comment (robhudson) [inline src/components/search/Search.svelte:3]: This PR is less fly

File: src/state/api.js [hunk 2]
```diff
@@ -58,24 +58,34 @@ export async function getProbeData(params, token) {
...
+
     const params = new URLSearchParams();
     const queryOptions = [];
     const stringScalars =
-      '(type.eq.scalar,definition->details->>kind.eq.string)';
+      '(type.eq.scalar,info->calculated->latest_history->details->>kind.eq.string)';
 
     queryOptions.push(`name.eq.${str}`);
-    queryOptions.push(`index.phfts(english).${str}`);
+    queryOptions.push(`search.phfts(simple).${str}`);
```
Comment (robhudson) [inline src/state/api.js]: `search.plfts(simple)` -- It's the difference of a phrase query or plain query. I think plain will be better suited here.

File: src/components/search/Search.svelte (hunk 1 above)
Comment (spasovski) [reply src/components/search/Search.svelte:3]: Not the right time to fly these days

**Comment by spasovski** [comment]
this got the go-ahead by @robhusdon via chat

**Pull request merged**